tests/integration-tests/tests/common/utils.py

```diff
@@ -90,11 +90,53 @@ def retrieve_latest_ami(region, os, ami_type="official", architecture="x86_64"):
         sys.exit(1)
 
 
+def retrieve_pcluster_ami_without_standard_naming(region, os, version="2.8.1", architecture="x86_64"):
```

Comment:
why are we hardcoding the version as a default?

Reply:
Yes, it is unnecessary now. I will remove them.